contxt/services/api.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         self.session = Session()
         self.session.auth = BearerTokenAuth(token_provider) if token_provider else None
         self.session.headers.update({"Cache-Control": "no-cache"})
+        self.session.headers.update({"User-Agent": get_http_user_agent()})
```

Comment:
this should be sufficient:
```suggestion
        self.session.headers.update({"User-Agent": f"contxt-sdk-python/{__version__}"})
```